qa/truffle/contracts/Storage.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
```

Comment:
Also what I've noticed is that we don't use consistent Solidity version. Here we are tolerating interval from 0.7.0 up to 0.9.0 in other places we are strict about 0.8.17 version and versions above version 0.8.0. Does that matter and should we use the same version in all places?

Reply:
Agree, whatever version it is - it should at least be consistent.